src/python_testing/TC_RR_1_1.py

```diff
@@ -163,11 +163,12 @@ async def test_TC_RR_1_1(self):
             fabrics: List[Clusters.OperationalCredentials.Structs.FabricDescriptorStruct] = await self.read_single_attribute(
                 dev_ctrl, node_id=self.dut_node_id, endpoint=0,
                 attribute=Clusters.OperationalCredentials.Attributes.Fabrics, fabricFiltered=False)
+            current_fabric_index = await self.read_single_attribute_check_success(cluster=Clusters.OperationalCredentials, attribute=Clusters.OperationalCredentials.Attributes.CurrentFabricIndex)
+            print(fabrics)
```

Comment:
do we want to keep this print in the test? It seems it is for debug mostly. Maybe logging.debug would be better.

Reply:
nah, it snuck in there when I was trying to figure out what the deuce was going on. Will remove.